build/configure.js

```diff
@@ -0,0 +1,156 @@
+'use strict';
+
+var utils = require('utils');
+// DependencyGraph is to manage build backend file, including generate and
+// execute.
```

Comment:
I prefer to put above this line of comment on the top of content in ./configure/dependencyGraph.js in case someone forgets to update this comment if logic of dependencyGraph.js is changed in someday.